primenet.py

```diff
@@ -5686,6 +5686,10 @@ def ping_server(ping_type=1):
     return None
 
 
+def is_pyinstaller():
+    return getattr(sys, 'frozen', False) and hasattr(sys, '_MEIPASS')
```

Comment:
This seems to be the canonical way to determine whether we're in a pyinstaller instance, so I've added it.

https://pyinstaller.org/en/stable/runtime-information.html

Reply:
For future reference, I added this link to a comment in the function.